sdk/include/ndk/ldrfuncs.h

```diff
@@ -97,8 +97,8 @@ LdrGetProcedureAddress(
 ULONG
 NTAPI
 LdrRelocateImage(
-    _In_ PVOID NewBase,
-    _In_ PCCH LoaderName,
+    _In_ PVOID BaseAddress,
+    _In_opt_ PCCH LoaderName,
```

Comment:
This LoaderName is a null-terminated string, so it should be: `PCSTR` (+ same comment for the LdrRelocateImageWithBias function)